modules/dxp/apps/portal-search-tuning/portal-search-tuning-gsearch-configuration-web/src/main/resources/META-INF/resources/js/components/Fragment.js

```diff
@@ -161,16 +146,14 @@ export default function Fragment({
 			</ClayList>
 
 			{expand && (
-				<div className="configuration-editor">
-					<AceEditor
-						onChange={handleChange}
-						onRender={({editorElement, editorTextInput}) => {
-							editorElementRef.current = editorElement;
-							editorTextInputRef.current = editorTextInput;
-						}}
-						value={value}
-					/>
-				</div>
+				<AceEditor
+					onBlur={(value) => setValue(value)}
+					onRender={({editorElement, editorTextInput}) => {
+						editorElementRef.current = editorElement;
+						editorTextInputRef.current = editorTextInput;
+					}}
```

Comment:
Is this needed anymore?

Reply:
Oh I took that from FieldMappings, I guess it's not necessary so I'll take it out!